### PR TITLE
CYF Glasgow Dalila Boumezrag |  Module-User-Focused-Data | week 2

### PR DESCRIPTION
**Pull request by DalilaBou**
Self checklist

- [ Done] I have committed my files one by one, on purpose, and for a reason
- [ Done] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ Done] I have tested my changes
- [ Done] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [Done ] My changes meet the [requirements](./README.md) of this task

Changelist

Questions

### CODE REVIEW

**Review by DalilaBou — COMMENTED** [review]
i fix it

**Comment by fcaroline2020** [comment]
Hi Dalila has this been reviewed already? It is missing the labels

File: Form-Controls/index.html [hunk 1]
```diff
@@ -7,21 +7,63 @@
...
+    <!-- write your html here-->
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
+
+    <div class="contact form">
+    <form >
+    <label for="Name">Your Name</label>
+    <input type="text" placeholder="Name">
```
Comment (SallyMcGrath) [inline Form-Controls/index.html:19]: What would happen if the customer did not add a name? What if the name they added was a number?

What do the criteria say about what the code needs to do?

File: Form-Controls/index.html [hunk 2]
```diff
@@ -7,21 +7,63 @@
...
+
+<label for="email"> Your Email</label>
```
Comment (SallyMcGrath) [inline Form-Controls/index.html:21]: When you tested your form, what happened when you submitted an invalid email? 

What about no email?

File: Form-Controls/index.html [hunk 3]
```diff
@@ -7,21 +7,63 @@
...
+  </select>
+
+
+    <label for="Date"> Date
+      <input type="date" max="2024-10-31"  min="2014-12-31" value="">
+    </label>
+    
+
+  </select>
+
```
Comment (SallyMcGrath) [inline Form-Controls/index.html:52]: How can the user submit this data?

File: Form-Controls/index.html [hunk 4]
```diff
@@ -7,21 +7,63 @@
...
+      <input type="date" max="2024-10-31"  min="2014-12-31" value="">
+    </label>
+    
+
+  </select>
+    
+  </form>
+</div>
+    
   </header>
```
Comment (SallyMcGrath) [inline Form-Controls/index.html:56]: What is this doing here? What does main mean?

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Hi @DalilaBou 

You have not added any validation attributes to your form, or a submit button. Please look again at the criteria. Go through each requirement and make sure you have actually met each one.

We really want to validate all data that we accept into a database. Revisit the prep materials, including the Q&A session, and have a proper think about this 

https://programming.codeyourfuture.io/user-data/sprints/2/prep/

**Pull request closed without merging**